src/view.ts

```diff
@@ -45,4 +45,64 @@ import { Frame } from "./types";
 
 export const view = (frames: Frame[]) => {
 
+    frames.map(makeFrameElements)
+
 }
+
+const makeFrameElements = (frame: Frame) => {
+
+    const container = document.createElement("div")
+    document.body.appendChild(container)
```

Comment:
can we review appendChilding? I think I'm hazy on why we do it / which parts do what to what. I think most specifically, why is this one document.body and the other ones container?

Reply:
Seconded!

Reply:
Let's say we wanted to create a display for the user that looks like this:
![image](https://user-images.githubusercontent.com/1444817/78946335-70870e80-7a90-11ea-98e5-d8c82c361854.png)

We've got four divs - One, Two, Three, and Four.

Note that One is a child of the document.body.

Note that Two, Three, and Four are all children of div One.

How do we get this hierarchy?  Like this!

``` javascript
document.body.appendChild(One)
One.appendChild(Two)
One.appendChild(Three)
One.appendChild(Four)
```